docs/docs/tutorials/orca-yarn-tutorial.md

```diff
@@ -7,33 +7,65 @@ This tutorial provides a step-by-step guide on how to run BigDL-Orca programs on
 A BigDL Orca program usually starts with the initialization of OrcaContext. For every BigDL Orca program, you should call `init_orca_context` at the beginning of the program as below:
 
 ```python
+# Create or get an OrcaContext on Spark Local
 from bigdl.orca import init_orca_context
 
-init_orca_context(...)
-```
-
-In `init_orca_context`, the user may specify necessary runtime configurations for the Orca program, including:
-
-* Cluster mode: Users can specify the computing environment for the program (a local machine, K8s cluster, Hadoop/YARN cluster, etc.), `init_orca_context` will create or get an OrcaContext with optimized configurations for BigDL performance, which will automatically provision the runtime Python environment and distributed execution engine.
-
-* Runtime: Users can specify the backend for the program (spark and ray, etc.) to create SparkContext and/or RayContext, the cluster mode would work based on the specified runtime backend.
+if cluster_mode == "local": # For local machine
+    init_orca_context(cluster_mode="local", cores=2, memory="2g", num_nodes=1)
+elif cluster_mode.startswith("yarn"): #  For Hadoop/Yarn cluster
+    if cluster_mode == "yarn-client":
+        init_orca_context(cluster_mode="yarn-client", cores=2, memory="10g", num_nodes=2, driver_cores=2, driver_memory="4g",
+                          extra_python_lib="/path/to/custom_module.py,/path/to/custom_module.zip", conf={"spark.driver.extraJavaOptions": "-Dbigdl.failure.retryTimes=1"})
+    elif cluster_mode == "yarn-custer":
+        init_orca_context(cluster_mode="yarn-cluster", cores=2, memory="10g", num_nodes=2, driver_cores=2, driver_memory="4g", 
+                          extra_python_lib="/path/to/custom_module.py,/path/to/custom_module.zip", conf={"spark.driver.extraJavaOptions": "-Dbigdl.failure.retryTimes=1"})
+elif cluster_mode == "bigdl-submit":
+    init_orca_context(cluster_mode="bigdl-submit")
+elif cluster_mode == "spark-submit":
+    init_orca_context(cluster_mode="spark-submit")
+```
```

Comment:
instead of showing example code here, just document the API, such as:
```python
   init_orca_context(cluster_mode, cores, memory, num_nodes, extra_python_lib, ...)
```